geoportal/geoportailv3_geoportal/static-ngeo/js/mymaps/mymaps.html

```diff
@@ -316,7 +316,7 @@ <h4 class="modal-title" translate>Ouvrir une carte</h4>
 
 
 <!-- Modification Modal -->
-<ngeo-modal ng-model="ctrl.modalShownHidden" ng-model-options="{getterSetter: true}">
+<ngeo-modal style="z-index:1050" ng-model="ctrl.modalShownHidden" ng-model-options="{getterSetter: true}">
```

Comment:
Should rebase and test with the fix: https://github.com/Geoportail-Luxembourg/geoportailv3/pull/3104/files before adding `style="z-index:1050"` to the modal